src/collections/integrations/aws-api-gateway-operator/index.mdx

```diff
@@ -11,8 +11,11 @@ featureList: ["Create, publish, maintain, monitor, and secure REST, HTTP, and We
 "API Gateway creates RESTful APIs that enable stateless client-server communication."
 ]
 workingSlides: [
-    ../_images/meshmap-visualizer.png,
-    ../_images/meshmap-designer.png]
+    ../_images/aws-api-Design.png,
+    ../_images/aws-api-Design-dark.png,
+    ../_images/aws-api-visualizer.png,
+    ../_images/aws-api-visualizer-dark.png
+    ]
```

Comment:
please revert this change after moving screenshots you've uploaded to its specific directory.
@Cvr421

Reply:
Refer to this: https://github.com/layer5io/layer5/pull/5129#issue-1995493031